docs/changes.rst

```diff
@@ -4,6 +4,12 @@ Change log
 Next version
 ------------
 
+* Changed caching monkey-patching for Django3.2+ to iterate over existing
+  caches and patch them individually rather than attempting to patch
+  ``django.core.caches`` as a whole. The ``middleware.cache`` is still
+  being patched as a whole in order to attempt to catch any cache
+  usages before ``enable_instrumentation`` is called. Fixes #1496
```

Comment:
```suggestion
  usages before ``enable_instrumentation`` is called.
```

Do we want to start adding ticket references? It may be useful but I'd rather not have to do this additional work :)

Reply:
You're right.